tasks/start.js

```diff
@@ -3,13 +3,40 @@
 var childProcess = require('child_process');
 var electron = require('electron-prebuilt');
 var gulp = require('gulp');
+var extend = require('util')._extend;
+var utils = require('./utils');
+
+var buildPath = './build';
 
 gulp.task('start', ['build', 'watch'], function () {
-    childProcess.spawn(electron, ['./build'], {
-        stdio: 'inherit'
-    })
-    .on('close', function () {
-        // User closed the app. Kill the host process.
-        process.exit();
+
+    utils.initBrowserSync({port: 3002}, function(bs) {
```

Comment:
I believe there is no sense in splitting this code into two places (here and utils file).
If you declare everything in `start.js` the code will also become a little simpler.